index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-gH2yIJqKdNHPEq0n4Mqa/HGKIhSkIHeL5AyhkYV8i59U5AR6csBvApHHNl/vI1Bx" crossorigin="anonymous">
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <nav id="navbar" class="navbar bg-dark nav justify-content-center">
+    <li class="nav-item">
+      <a class="nav-link navbar-brand" href="#">Portfolio</a>
+    </li>
+    <li class="nav-item">
+      <a class="nav-link" href="#">Next</a>
+    </li>
+    <li class="nav-item active">
+      <a class="nav-link" href="#">Home</span></a>
+    </li>
+    <li class="nav-item">
+      <a class="nav-link" href="#projects">Projects</a>
+    </li>
+  </nav>
+  <section id="welcome-section">
```

Comment:
- [ ] [optional] Please note that the navbar should be within a header tag :smiley: . More information about it can be found [here](https://www.geeksforgeeks.org/html5-header-tag/#:~:text=The%20tag%20in%20HTML,but%20this%20is%20not%20required.)

Reply:
- [ Optional] Also, you did great providing a link it would be great to explain why in detail. The links are only to provide additional info and to fill any skill gap you think there is. You need to make your suggestion as clear and polite as possible. This will result in better collaboration in a team and keep the vibe positive.